flink-table-store-core/src/main/java/org/apache/flink/table/store/file/mergetree/compact/UniversalCompaction.java

```diff
@@ -63,7 +63,8 @@ public UniversalCompaction(int maxSizeAmp, int sizeRatio, int maxRunNum) {
         // 3 checking for file num
         if (runs.size() > maxRunNum) {
```

Comment:
Trigger number for this compaction and other compactions should be different (see [rocksdb wiki](https://github.com/facebook/rocksdb/wiki/Universal-Compaction#4-compaction-triggered-by-number-of-sorted-runs-without-respecting-size_ratio)). Add a new option.

Reply:
Sorry for misleading. They're actually the same option.